mithril-client-wasm/www-test/package-lock.json

```diff
@@ -21,7 +21,7 @@
     },
     "../pkg": {
       "name": "mithril-client-wasm",
-      "version": "0.5.2",
+      "version": "0.5.3",
```

Comment:
This [commit](https://github.com/input-output-hk/mithril/pull/2004/commits/2ba740722ec3214421f9e7ee48db9a2a0849f02e) shouldn't have modified the version of the `mithril-client-wasm` dependency. Only the version of the JavaScript package should have been updated.

Reply:
Also, the commit message should have been something like:
`chore: bump Javascript packages versions`
instead of:
`chore: bump mithril client wasm 'www' and 'www-test' dependencies`